packages/cli-debugger-ui/src/ui/index.html

```diff
@@ -14,6 +14,27 @@
   </head>
   <body>
     <div class="content">
+      <div class="warning">
+        <p>
+          Remote JavaScript debugging (this workflow) is
+          <strong>deprecated</strong> in React Native 0.73 and will be removed
+          in React Native 0.74.
+        </p>
+        <p>
+          Please use the
+          <strong>Open Debugger</strong> workflow to debug apps using Hermes
```

Comment:
@motiz88 One thought here is whether we should mention or link any info on future JSC direct debugging capabilities — i.e. pointing to an umbrella issue on GitHub that we open during 0.73's lifetime, to gather signal on users who may need this. This issue would reinforce that JSC debugging is no longer directly supported, but that we're looking for interested owners to implement JSC support in the new debugging stack.